kernel/kernel-api/src/main/java/io/delta/kernel/Scan.java

```diff
@@ -106,6 +110,16 @@ public interface Scan {
      */
     Row getScanState(TableClient tableClient);
 
+    /**
+     * Get the physical data read schema to pass to the parquet handler to read the given data file.
```

Comment:
```suggestion
     * Get the physical data read schema to read the  data from given scan file.
```

Reply:
Parquet handler is not required to be used by the connector. They can have their own Parquet reader.